WD-Firmware/examples/ex-freertos/main.c

```diff
@@ -243,23 +247,24 @@ void demo_init(void *pMem)
     can be done here if it was not done before main() was called. */
     prvSetupHardware();
 
+// NatiR: what happens when we are not in D_USE_RTOSAL? where the ISRs are installed?
 #ifdef D_USE_RTOSAL
     /* Disable the machine & timer interrupts until setup is done. */
     clear_csr(mie, MIP_MEIP);
     clear_csr(mie, MIP_MTIP);
     /* register exception handlers */
     for (cause = E_EXC_INSTRUCTION_ADDRESS_MISALIGNED ; cause < E_EXC_LAST ; cause++)
     {
-        pspRegisterIsrExceptionHandler(vSynchTrapUnhandled, cause);
+        pspRegisterIsrExceptionHandler(pspTrapUnhandled, cause);
     }
     /* register E_CALL exception handler */
-    pspRegisterIsrExceptionHandler(vSynchTrap, E_EXC_ENVIRONMENT_CALL_FROM_MMODE);
+    pspRegisterIsrExceptionHandler(pspEcallHandler, E_EXC_ENVIRONMENT_CALL_FROM_MMODE);
     /* install timer interrupt handler */
-    pspRegisterIsrCauseHandler(vPortSysTickHandler, E_MACHINE_TIMER_CAUSE);
+    pspRegisterIsrCauseHandler(pspMTimerIntHandler, E_MACHINE_TIMER_CAUSE);
     /* install external interrupt handler */
     pspRegisterIsrCauseHandler(handle_interrupt, E_MACHINE_EXTERNAL_CAUSE);
     /* Enable the Machine-External bit in MIE */
-    set_csr(mie, MIP_MEIP);
+    set_csr(mie, MIP_MEIP); // NatiR: why not re-enable MIP_MTIP too?

```

Comment:
remove the comment

Reply:
Irrelevant as this file is being abandoned

Reply:
remove the ex-freertod directory + main.c

Reply:
removed